src/schema/rules/checks/meg.yaml

```diff
@@ -0,0 +1,14 @@
+# Rules for MEG data that cannot be defined in tables
+---
+MEGElectrodesRecommended:
```

Comment:
as outlined in https://github.com/bids-standard/bids-specification/issues/1550#issuecomment-1686032272, I think this should be optional --> this electrode information **can** also be stored in `_headshape.<ext>` files or sometimes in the data file format itself (Elekta/Neuromag/MEGIN).